packages/assets/scss/00-base/_colors.scss

```diff
@@ -46,3 +45,8 @@ $c-bd-divider-dark:      $c-gray-dark !default;
 $c-bd-error:             $c-error !default;
 $c-bd-input:             $c-gray-light !default;
 $c-bd-input-dark:        $c-gray !default;
+
+// Box shadow
+$c-box-shadow-black:      rgba($c-black, 0.25) !default;

```

Comment:
I think this shadow setting is used commonly enough that deserves its own variable `0 0.25rem 0.5rem $c-box-shadow-black;` Can you add a variable to store the box-shadow value as a whole? Thanks